debian/control

```diff
@@ -44,7 +44,7 @@ Depends:
  python3-psycopg2,
  python3-pydot,
  python3-openssl,
- python3-pypdf2,
+ python3-pypdf2 | python3-pypdf,
```

Comment:
Actually I noticed that in deb_install (and docker image) this will have as side effect to have both dependencies installed.

Reply:
It should be handled properly with this PR 🤔

Reply:
Odoo will handle it but it also mean that since python3-pypdf will be present in the docker image, it will be easier to introduce code that would work by mistake in 18, using only python3-pypdf and not python3-pypdf2.
In trixie we will only have python3-pypdf but in noble we will have both versions.

Reply:
[pypdf2 is tried first ](https://github.com/odoo/odoo/pull/183165/files#diff-ea6d0f8ec9f32a16a9f9bb6ecec9aec0d5e98be461042a3aacd854ee5a26b7bdR29)

Reply:
in debian controll yes but with apt no